### PR TITLE
Feature/craf 24 scalar function

### PR DESCRIPTION
**Pull request by EdwinToTheBatCave**
Scalar function to count kegs not sold for a particular batch

### CODE REVIEW

File: sql/functions.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
+--liquibase formatted sql
+
+--changeset emokoena:create-count-unsold-kegs-from-batch
+CREATE FUNCTION CountUnsoldKegsFromBatch(batch_id INT)
+RETURNS INT
```
Comment (ryanbasiltrickett) [inline sql/functions.sql]: Use integer (lower case) for variable types

**Review by ryanbasiltrickett — CHANGES_REQUESTED** [review]
I believe this function should be on a specific beer (name) and not a batchID

File: sql/functions.sql (hunk 1 above)
Comment (EdwinToTheBatCave) [reply sql/functions.sql]: Cool On it

**Comment by EdwinToTheBatCave** [comment]
> I believe this function should be on a specific beer (name) and not a batchID

My thinking is that a beer name isn't a unique identifier

**Pull request merged**